cmd/sys-mgmt-agent/res/configuration.toml

```diff
@@ -16,9 +16,10 @@ ServiceAddress = 'localhost'
 CheckInterval = '10s'
 EnableRemoteLogging = false
 LoggingFile = './logs/edgex-sys-mgmt-agent.log'
-# OperationsType = 'os'
-OperationsType = 'docker'
-ComposeUrl = 'https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/compose-files/docker-compose-delhi-0.7.0.yml'
+
+ExecutorPath = '/path/to/the/file'
+# The following (ExecutorPath) is DEBUG_ONLY:
+# ExecutorPath = '/Users/akramahmad/go/src/github.com/edgexfoundry/docker-compose-executor/main'
```

Comment:
I still would prefer this to not be here, but it's at least commented out so :man_shrugging: